testsuite/test_image_classification.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+from snoop.data.analyzers import image_classification
+from conftest import TESTDATA, CollectionApiClient
+
+pytestmark = [pytest.mark.django_db]
+
+TEST_IMAGE = TESTDATA / './disk-files/images/bikes.jpg'
+
+
+def test_classification_service():
+    with TEST_IMAGE.open('rb') as f:
+        image_classification.call_image_classification_service(f, 'bikes.jpg')
```

Comment:
please check the results in this test, or at least check for some important classes